### DISCUSSION TITLE
What version of Saleor is in the NextJS Commerce?

### DISCUSSION THREAD

**Discussion by RyanTamulevicz** [saleor/saleor#9479] [Deployment]
At https://saleor.vercel.store/ which version of saleor is being used here? I tried to query the graphql used for that site and I did not have permission to check through that. I was curious if anyone knew. I am trying to find a way to get a site up and running ASAP for production so any storefront that is fully functioning is what I need right now then I will worry about changes after it is up...

I ask because I am getting this error when using my own Saleor api vs the one provided on the example site (error on 24): 

> TypeError: Cannot read property 'products' of null
  22 |         if (featured) {
  23 |             var ref, ref1;
> 24 |             const products = ((ref = data.collection.products) === null || ref === void 0 ? void 0 : (ref1 = ref.edges) === null || ref1 === void 0 ? void 0 : ref1.map(({ node: p  })=>normalizeProduct(p)
  25 |             )) ?? [];
  26 |             return {
  27 |                 products

This is also the case with the Saleor demo graphql

**Comment by karolkielecki** [comment]
Hi @RyanTamulevicz, the error above is related to the fronted side--it's trying to get the products field. I'm not sure which Saleor version is used at https://saleor.vercel.store/. Let me ask the team, and I'll get back to you soon. 

We're working on a new project--have you seen https://github.com/saleor/react-storefront? However, please bear in mind that it's a starter kit.

**Reply by RyanTamulevicz** [reply]
I have! I am waiting for a few more things like the checkout to be further along from what I was reading. Was mostly looking for something to get out ASAP and then I would do my own design using the starter kit.  

Thank you very much for all your work on Saleor by the way!
